PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/common/HDInsightUtil.java

```diff
@@ -165,8 +165,10 @@ public static void showWarningMessageOnSubmissionMessageWindow(@NotNull final Pr
     private static void showInfoOnSubmissionMessageWindow(@NotNull final Project project, @NotNull final MessageInfoType type, @NotNull final String message, @NotNull final boolean isNeedClear) {
 
         synchronized (LOCK) {
-            final ToolWindow toolWindow = ToolWindowManager.getInstance(project).getToolWindow(CommonConst.SPARK_SUBMISSION_WINDOW_ID);
-            showSubmissionMessage(toolWindow, project, message, type, isNeedClear);
+            try {
+                final ToolWindow toolWindow = ToolWindowManager.getInstance(project).getToolWindow(CommonConst.SPARK_SUBMISSION_WINDOW_ID);
+                showSubmissionMessage(toolWindow, project, message, type, isNeedClear);
+            } catch (Exception ignore) { /* IDE disposed */ }
```

Comment:
I hit the issue once closing the Intellij, there was a AssertionError "Already disposed" thrown from `ToolWindowManager.getInstance(project)`